### PR TITLE
First pass: Skip upload of Application Version if one already exists …

### PR DESCRIPTION
**Pull request by danieljimenez**
…with the same version number

I'm open to refactoring / renaming whatever. I did this in a bit of a coffee induced lack of sleep 😄 .

Issue #7

### CODE REVIEW

File: aws-elasticbeanstalk-common/src/main/java/jetbrains/buildServer/runner/elasticbeanstalk/AWSClient.java [hunk 1]
```diff
@@ -242,6 +263,10 @@ void createVersionFinished(@NotNull String applicationName, @NotNull String vers
                                @NotNull String s3BucketName, @NotNull String s3ObjectKey) {
     }
 
+    void createVersionSkipped(@NotNull String applicationName, @NotNull String versionLabel) {
```
Comment (danieljimenez) [inline aws-elasticbeanstalk-common/src/main/java/jetbrains/buildServer/runner/elasticbeanstalk/AWSClient.java:266]: False positive? This is how the other methods are in the class.

Comment (danieljimenez) [inline aws-elasticbeanstalk-common/src/main/java/jetbrains/buildServer/runner/elasticbeanstalk/AWSClient.java:266]: Same as below.

Comment (rtfpessoa) [reply aws-elasticbeanstalk-common/src/main/java/jetbrains/buildServer/runner/elasticbeanstalk/AWSClient.java:266]: I setup static analysis but did not got time to tweak it. Do not worry for now.

**Comment by danieljimenez** [comment]
#8

**Comment by rtfpessoa** [comment]
@danieljimenez can you amend the last commit to see if Circle CI triggers the build?

**Comment by rtfpessoa** [comment]
@danieljimenez have you seen my comment?

**Comment by danieljimenez** [comment]
Oops, no let me take a look!

**Comment by danieljimenez** [comment]
Same thing it seems like.

**Comment by rtfpessoa** [comment]
@danieljimenez now we have circle ci status. LGTM :+1:

**Pull request merged**